### DISCUSSION TITLE
502 Error

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#384] [Q&A] [answered]
Im getting 502 error when odoo is calling endpoints:
/longpolling/poll
longpolling/im_status
POST https://somedomain.com/longpolling/poll 502

What can cause this behaviour?
In odoo interface connection lost and connection restore pop-up shows every 2 seconds.

I dont know if this is related but in traefik log i can see:

ERRO[2021-02-06T06:53:55Z] Provider connection error unexpected EOF, retrying in 276.473075ms

**Comment by rquevedo** [comment]
@joao-p-marques any idea of this?
Im running doodba in prod mode.

**Reply by joao-p-marques** [reply]
Hi!

What are the contents of the `domains_prod` variable? 

Maybe @Yajo can help here.

**Comment by yajo** [comment, marked as answer]
You seem to have some misconfiguration around at least one of these things:

1. Odoo not launched with workers.
2. Traefik cannot connect properly to docker.
3. Traefik badly configured.
4. Bad answers to `domains_prod` and/or `domains_test`.

All of these are aspects specific to your deployment, so you will have to review those points.

**Reply by rquevedo** [reply]
Hi @Yajo, you got it right on your first try. Odoo was not launched with workers. The thing was that i do many test using doodba in TEST mode and in this mode odoo server starts with --workers 2 by default but in prod let you configure properly.
Thanks again to you and to @joao-p-marques
